kaitai_utils.py

```diff
@@ -90,10 +93,13 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
                 elif _is_list(value_object):
                     yield _to_lower_camel_case(key), self._list_to_dict(value_object, path)
                 elif isinstance(value_object, bytes):
-                    if len(value_object) < hansken_extraction_plugin.runtime.constants.MAX_CHUNK_SIZE:
-                        child_builder = self.trace.child_builder(path)
-                        child_builder.update(data={'raw': value_object}).build()
-                    yield _to_lower_camel_case(key), "data block of size: " + str(len(value_object))
+                    if len(value_object) > self.max_byte_array_length:
+                        yield _to_lower_camel_case(key), "data block of size: " + str(len(value_object))
```

Comment:
perhaps explicitly mention that data was added as a child trace?
e.g. data block of size: 123 (stored as child trace $my.magic.path)

(and if the data was not added as a child, mention this as well?)

Reply:
Done, also added some text when the size of the data exceeds the limit.